lib/nostr.js

```diff
@@ -16,6 +16,11 @@ export function hexToBech32 (hex, prefix = 'npub') {
   return bech32.encode(prefix, bech32.toWords(Buffer.from(hex, 'hex')))
 }
 
+export const truncateNpub = (string = '', maxLength = 140) =>
+  string.length > maxLength
+    ? `${string.substring(0, maxLength)} […]`
+    : string
+
```

Comment:
Doesn't seem to be used

_didn't pull the code, only used CTRL+F in the browser_